src/pagesComponents/Auth/LeftPanel/Form.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+import {
+  Form, FormField, SubmitButton, LoginButton,
+} from '../../../components';
+import { loginSchema, registerSchema } from '../../../utils/validation';
+import { LoginDiv, RegisterDiv } from './styles';
+
+export default function FormAuth() {
+  const navigate = useNavigate();
+  const url = window.location.search.substring(1);
+  const { loginWithRedirect } = useAuth0();
+  let initialValues = {};
+  let validationSchema = {};
+
+  if (url === 'register') {
```

Comment:
Do not manually check for URLs, use React Router to get the current URL and create a map of initialValues + validationSchemas for every URL.